dora/src/vm/namespaces.rs

```diff
@@ -48,7 +50,12 @@ impl NamespaceData {
         }
     }
 
-    pub fn new(vm: &mut VM, parent_id: NamespaceId, ast: &Arc<Namespace>) -> NamespaceData {
+    pub fn new(
+        vm: &mut VM,
+        file_id: FileId,
```

Comment:
Not sure this makes sense atm, namespaces can span multiple files (a whole directory) at the moment, so there isn't a single file a namespace is defined in.